xcube_geodb/core/user.py

```diff
@@ -0,0 +1,318 @@
+# coding: utf-8
+import os
+from abc import abstractmethod
+from typing import Dict, Optional
+from urllib.parse import urljoin
+import logging
+from xml.etree.ElementTree import XML
+import requests
+from dotenv import find_dotenv, load_dotenv
+from geoserver import settings
+
+from xcube_geodb.defaults import GEOSERVER_DEFAULT_PARAMETERS
+
+LOGGER = logging.getLogger("gsconfig2.catalog")
+
+
+class GeoserverSession:
+    """
+    The GeoServer catalog represents all of the information in the GeoServer
```

Comment:
Class is named `GeoserverSession` but docs say "GeoServer catalog". Please clarify.
Also explain the purpose of the class.

Reply:
Why is it in a module called `user`?

Reply:
Same as for class user.